.github/scripts/get-commit-from-ref.sh

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env bash
+
```

Comment:
should we set errexit? We have the validation up in JS but I can picture somehow this outputs a valid string but didn't actually `cd` or something to get it